lib/resources/datav2/websockets_v2.js

```diff
@@ -183,12 +183,14 @@ class AlpacaStreamV2Client extends events.EventEmitter {
   }
 
   subscribeAll() {
+    const { trades, quotes, bars, dailyBars, statuses } =
+      this.session.subscriptions;
     if (
-      this.session.subscriptions.trades.length > 0 ||
-      this.session.subscriptions.quotes.length > 0 ||
-      this.session.subscriptions.bars.length > 0 ||
-      this.session.subscriptions.dailyBars.length > 0 ||
-      this.session.subscriptions.statuses.length > 0
+      trades.length ||
+      quotes.length ||
+      bars.length ||
+      dailyBars.length ||
+      statuses.length
```

Comment:
You can still include the > 0. More precise more readable in this case I think. 
Or if you want to code as less as possible you can omit the `.length` I think